packages/r/react-spring.json

```
@@ -8,6 +8,7 @@
     "animation",
     "spring"
   ],
+  "filename": "konva.cjs.d.ts",
```

```suggestion
  "filename": "web.cjs.js",
```
Let's suggest the most generic version they supply